src/assets/crubrand/generic/mixins/_media-query.scss

```diff
@@ -17,13 +19,13 @@ $query-unit: 16px;
             @content;
         }
    
-        @media screen and (#{$query1}-#{$query2}: strip-units($point) / strip-units($query-unit) + em) {
+        @media screen and (#{$query1}-#{$query2}: list.slash(strip-units($point), strip-units($query-unit) + em)) {
```

Comment:
What does `list.slash` do? This looks like a division...

Reply:
Yes, it should behave the same way as a division.  I got an error saying that `math.div` should only be used for number variables, and I guess it didn't know how to process the results of `strip-units`.  then again, `strip-units` is used in other places and it doesn't seem to cause an issue, I'll have to look into that.

I'm actually noticing another order-of-operations issue here -- I assumed that `+ em` was part of the right side of the `/` operator, but now I realize that it should be it's own thing.